synergy-design-system/synergy-design-system
# feat: ✨ syn-alert

# Pull Request

## 📖 Description

This PR adds the new `<syn-alert />` component.

### 🎫 Issues

Closes #377 

## 👩‍💻 Reviewer Notes

- This version of the alert does **not** provide the `helpers/alert/<slot>` functionality that makes it possible to draw a button on the bottom right of the screen. This would either need to be a template or its own component. Needs clarification (@reichst1SICKAG)
- As borders in html get connected at their ankles, I was not able to set a `border-left` of 5px and a `border-top` of 1px without visual glitches (e.g. triangle building). The `border` styling is therefore done via a custom css `:after` property inserted into the DOM. The spacing of the top left most element are automatically adjusted to include this virtual `border-left-width`.

## 📑 Test Plan

- Check out this branch and build world
- Start storybook and inspect the stories

## ✅ DoD
